ksql-engine/src/test/java/io/confluent/ksql/util/FakeKafkaTopicClient.java

```diff
@@ -52,6 +56,16 @@ public int getNumPartitions() {
     public short getReplicatonFactor() {
       return replicatonFactor;
     }
+
+    public TopicDescription getDescription() {
+      List<TopicPartitionInfo> partitionInfoList = new LinkedList<>();
+      Node node = new Node(0, "localhost", 9091);
+      for (int i = 0; i < numPartitions; i++) {
+        partitionInfoList.add(
+            new TopicPartitionInfo(0, node, new LinkedList<>(), new LinkedList<>()));
```

Comment:
Can these `new LinkedList`s just be `Collections.emptyList()`, or do they need to be mutable?

Reply:
Nope. Still learning the latest and greatest java idioms :)